### PR TITLE
Randyh0120

### PR DESCRIPTION
**Pull request by randyh0120**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1891"></a>

### CODE REVIEW

File: public/data/randyh0120.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "Randy Hattab",
+  "type": "personal",
+  "bio": "Software Engineering FGCU 2020, MBA FGCU 2023  | Full-Stack & Mobile Software Engineer",
```
Comment (Panquesito7) [inline public/data/randyh0120.json:4]: I recommend removing this little space here. What do you think?
```suggestion
  "bio": "Software Engineering FGCU 2020, MBA FGCU 2023 | Full-Stack & Mobile Software Engineer",
```

File: public/data/randyh0120.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      "title": "SWE @ ATP",
+      "date": "March 2021",
+      "icon": "plane",
+      "color": "green",
+      "description": "Working on Maintenance and Operations software for the aviation industry"
+    },
+    {
+      "title": "SWE @ Lee Health",
+      "date": "March 2020",
+      "icon": "racecar",
```
Comment (EmmaDawsonDev) [inline public/data/randyh0120.json:46]: None of these icons exist in our repo so they will fall back to default. Take a look at the list of available icons in icons.md

File: public/data/randyh0120.json (hunk 1 above)
Comment (randyh0120) [reply public/data/randyh0120.json]: Updated and fixed

File: public/data/randyh0120.json (hunk 2 above)
Comment (randyh0120) [reply public/data/randyh0120.json]: Updated and fixed

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**